modules/gnu-utility/init.zsh

```diff
@@ -15,16 +15,16 @@ fi
 
 _gnu_utility_cmds=(
   # Coreutils
-  '[' 'base64' 'basename' 'cat' 'chcon' 'chgrp' 'chmod' 'chown'
+  '[' 'b2sum' 'base32' 'base64' 'basename' 'cat' 'chcon' 'chgrp' 'chmod' 'chown'
   'chroot' 'cksum' 'comm' 'cp' 'csplit' 'cut' 'date' 'dd' 'df'
   'dir' 'dircolors' 'dirname' 'du' 'echo' 'env' 'expand' 'expr'
   'factor' 'false' 'fmt' 'fold' 'groups' 'head' 'hostid' 'id'
-  'install' 'join' 'kill' 'link' 'ln' 'logname' 'ls' 'md5sum'
-  'mkdir' 'mkfifo' 'mknod' 'mktemp' 'mv' 'nice' 'nl' 'nohup' 'nproc'
-  'od' 'paste' 'pathchk' 'pinee' 'pr' 'printenv' 'printf' 'ptx'
+  'install' 'join' 'kill' 'link' 'ln' 'logname' 'ls' 'md5sum' 'mkdir'
+  'mkfifo' 'mknod' 'mktemp' 'mv' 'nice' 'nl' 'nohup' 'nproc'
+  'numfmt' 'od' 'paste' 'pathchk' 'pinky' 'pr' 'printenv' 'printf' 'ptx'
```

Comment:
It looks like `pinee` was replaced with `pinky`... I can't find a reference to either of these in GNU coreutils... do you have any background info on this?

Reply:
No particular background, just the list that comes from `ls /usr/local/opt/coreutils/bin` on mac os

```
~ $ ls /usr/local/opt/coreutils/bin
'g['         gdf          gjoin      gpaste       gsha512sum   gtrue
 gb2sum      gdir         gkill      gpathchk     gshred       gtruncate
 gbase32     gdircolors   glink      gpinky       gshuf        gtsort
 gbase64     gdirname     gln        gpr          gsleep       gtty
 gbasename   gdu          glogname   gprintenv    gsort        guname
 gcat        gecho        gls        gprintf      gsplit       gunexpand
 gchcon      genv         gmd5sum    gptx         gstat        guniq
 gchgrp      gexpand      gmkdir     gpwd         gstdbuf      gunlink
 gchmod      gexpr        gmkfifo    greadlink    gstty        guptime
 gchown      gfactor      gmknod     grealpath    gsum         gusers
 gchroot     gfalse       gmktemp    grm          gsync        gvdir
 gcksum      gfmt         gmv        grmdir       gtac         gwc
 gcomm       gfold        gnice      gruncon      gtail        gwho
 gcp         ggroups      gnl        gseq         gtee         gwhoami
 gcsplit     ghead        gnohup     gsha1sum     gtest        gyes
 gcut        ghostid      gnproc     gsha224sum   gtimeout     realpath
 gdate       gid          gnumfmt    gsha256sum   gtouch
 gdd         ginstall     god        gsha384sum   gtr
```